### PR TITLE
fix: Updated Ticket Mobile UI

### LINKED ISSUE #5915: Tickets: fix mobile ticket view
Screenshot_20201205_084459_com android chrome

### PR DESCRIPTION
**Pull request by SARTHAK1SINGH**
Fixes #5915

#### Short description of what this resolves:
Updated the UI for the mobile view as Told in the issue.

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by SARTHAK1SINGH** [comment]
## New UI of the mobile view of Ticket. <br>
Screenshot 2020-12-16 165209

File: app/templates/components/public/ticket-list.hbs [hunk 1]
```diff
@@ -1,5 +1,227 @@
...
+                </tr>
+              </tbody>
+            </table>
+        </td>
+      </tr>
+    </tfoot>
+    </table>
+
+{{!-- order now --}}
+    {{#unless this.amountEditable}}
```
Comment (maze-runnar) [inline app/templates/components/public/ticket-list.hbs:152]: is it needed to repeat code for order button, please remove this. Keep it out side if else.

**Review by daretobedifferent18 — COMMENTED** [review]
Screenshot from 2020-12-16 20-04-34
I don't think this could be nice way of showing heading on left and value on right for each ticket.
What say? @mariobehling

File: app/templates/components/public/ticket-list.hbs (hunk 1 above)
Comment (SARTHAK1SINGH) [reply app/templates/components/public/ticket-list.hbs:152]: Ok, I'll do the changes.

**Comment by iamareebjamal** [comment]
This is massive code duplication

**Comment by daretobedifferent18** [comment]
> This is massive code duplication

Exactly, we need to implement the way where duplication is not there

**Pull request closed without merging**